#Standard header
Header header

#######################
# Debug 50-53
#######################
uint16 register_50
uint16 register_51
uint16 register_52
uint16 register_53

